### PR TITLE
Normalized Copyright notices

### PR DESCRIPTION
**Pull request by rozza**
JAVA-2755

### CODE REVIEW

File: bson/src/main/org/bson/codecs/AtomicBooleanCodec.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 MongoDB, Inc.
+ * Copyright 2008-present MongoDB, Inc.
```
Comment (jyemin) [inline bson/src/main/org/bson/codecs/AtomicBooleanCodec.java]: This should be 2015-present

File: config/checkstyle.xml [hunk 2]
```diff
@@ -33,12 +33,12 @@
...
           <pre>
             /*
-             * Copyright 2008-2014 MongoDB, Inc.
+             * Copyright 2008-present MongoDB, Inc.
              *
           </pre>
         -->
         <property name="format"
-                  value="^(//| \*) Copyright (\([cC]\) )?[\d]{4}(\-[\d]{4})? (MongoDB, Inc\.).*$" />
+                  value="^(//| \*) Copyright 2008-present MongoDB, Inc\.$" />
```
Comment (jyemin) [inline config/checkstyle.xml]: I didn't read the drivers ticket as starting every file at 2008.

Comment (rozza) [reply config/checkstyle.xml]: I did, the copyright is for the project as a whole, source and docs. It also makes it much easier as well.

Comment (jyemin) [reply config/checkstyle.xml]: Yeah, I see what you mean after re-rereading the ticket.  But it was interpreted differently in PyMongo: https://github.com/mongodb/mongo-python-driver/commit/507f954ed4f5778b2cc2d34fd57ff5525f4d2be0.  I'll ask Bernie to take a look.

File: bson/src/main/org/bson/codecs/AtomicBooleanCodec.java (hunk 1 above)
Comment (jyemin) [reply bson/src/main/org/bson/codecs/AtomicBooleanCodec.java]: Never mind

File: config/checkstyle.xml (hunk 2 above)
Comment (jyemin) [reply config/checkstyle.xml]: Confirmed that this way is fine.

**Pull request closed without merging**